desktop_version/src/Screen.cpp

```diff
@@ -170,7 +170,7 @@ void Screen::ResizeScreen(int x, int y)
 	if (stretchMode == 1)
 	{
 		int winX, winY;
-		SDL_GetWindowSize(m_window, &winX, &winY);
+		SDL_GetRendererOutputSize(m_renderer, &winX, &winY);
```

Comment:
This call should use `Screen::GetWindowSize()` like all other instances of this call.